its/ruling/src/integrationTest/resources/expected/azureresourcemanager/azureresourcemanager-S6388.json

```diff
@@ -6,22 +6,28 @@
 "azureresourcemanager-project:ruling/src/integrationTest/resources/sources/azureresourcemanager/rules/sas9.4-viya-azuredeploy.json": [
 628,
 1447,
-1452,
```

Comment:
It is still raising, just not on the same "level." The 1452 was highlighting the `managedDisk` directly because it was only rising when `managedDisk` existed. 

The issues are now raised on the whole disk (`osDisk`here), just because there are multiple ways of fixing it and we're not sure that `managedDisk`exist when we're raising.

At some point, I tried to have some more logic. Instead of highlighting the whole disk, highlight only the `managed disk` when it already exists. But there are so many edge cases and possibilities that the logic would be way harder to implement.

Same explanation for other similar issues.